contrib/core/actions/uuid.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: uuid
```

Comment:
So I had this same conversation in my head at the time. My descision was to leave it uuid to allow for a a `type` parameter to be added later on. This would default to uuid4 but would allow the user choose, say uuid1 all in the same action. Thoughts on that? Also ping @lakshmi-kannan